src/main/java/dan200/computercraft/core/apis/FSAPI.java

```diff
@@ -268,6 +377,14 @@ public final void delete( String path ) throws LuaException
         }
     }
 
+    /**
+     * Returns the amount of free space available on the drive the path is
+     * located on.
+     *
+     * @param path The path to check the free space for.
+     * @return The amount of free space available, in bytes.
```

Comment:
Can you add a `@cc.treturn number|"unlimited"` here. And possibly explain the whole "unlimited" thing.

Reply:
I added this, but I'm not really sure what causes `"unlimited"` to be returned. I think I've seen it before but I can't remember what caused it. Could you explain what would cause this? I see it's returned if the underlying FS's `getRemainingSpace` returns -1, but there doesn't seem to be an example of this in any implementations of `IWritableMount`.

Reply:
So this code path is never hit using CC's builtin mounts, but I suppose it exists in the unlikely case that an external mod adds a mount without any file size restrictions? I'm afraid this is Dan code, so I can't really comment on the thinking behind it :).